CODE_REVIEW_CHECKLIST.md

```diff
@@ -69,3 +71,10 @@ may have.
 
 - [ ] Will this change work on all of our supported platforms and
```

Comment:
Good catch -- While I'd say _"everything we build for in Jenkins"_, this is not something an outside contributor would be able to see. I'll update this.